app/assets/javascripts/hyrax/app.js

```diff
@@ -134,5 +135,12 @@ Hyrax = {
     statistics: {},
 
     // initialized in hyrax/config.js
-    config: {}
+    config: {},
+
+    // Per Page select will submit its form to change records shown
+    batchSubmit: function () {
+        var BatchSubmit = require('hyrax/batch_select');
```

Comment:
Why are we calling the file `batch_select` but the constant `BatchSubmit`? Can we decide on submit or select and stick with it?